### DISCUSSION TITLE
Demande de liste des composants électroniques pour le système Mycodo / Request for a list of electronic components for the Mycod

### DISCUSSION THREAD

**Discussion by bilel-koubaa** [kizniche/Mycodo#1405] [Q&A]
Bonjour,
Je suis très satisfait de Mycodo. Étant nouveau dans ce domaine, je souhaite mettre en place le système de contrôle Mycodo. Pourriez-vous, s'il vous plaît, me fournir la liste complète des composants électroniques nécessaires, avec leurs références, afin de reproduire le même système publié par Mycodo ?
Merci d'avance.
Hello, 
I am very pleased with Mycodo. As I am new to this field, I would like to set up the Mycodo control system. Could you please provide me with the complete list of required electronic components, along with their references, to replicate the same system published by Mycodo? 
Thank you in advance.

**Comment by jimbo762** [comment]
I think that list is on the back of the receipt.
